src/shared/components/palette/complex/ccd-collection-table-value-case-fields.pipe.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { CcdCollectionTableCaseFieldsFilterPipe } from './ccd-collection-table-value-case-fields.pipe';
+
+describe('CddFieldsKeyValueCaseFieldPluckPipe', () => {
```

Comment:
Why does this say PluckPipe?

Reply:
Good spot I renamed the pipe guess It didn't change the string value